test/functional/ui/screen.lua

```diff
@@ -1146,7 +1160,9 @@ function Screen:redraw_debug(attrs, ignore, timeout)
   local function notification_cb(method, args)
     assert(method == 'redraw')
     for _, update in ipairs(args) do
-      print(require('inspect')(update))
+      if update[1] ~= "mode_info_set" then
```

Comment:
is this because mode_info_set is noisy? Would be good to add a comment.

Reply:
indeed.